container/shim/src/fetchers/lassie.js

```diff
@@ -115,7 +122,7 @@ export async function respondFromLassie(req, res, { cidObj, format }) {
     pipeline(lassieRes.body, metricsStream, () => {});
 
     if (isRawFormat) {
-      await getRequestedBlockFromCar(metricsStream, res, cidObj, blockFilename);
+      await getRequestedBlockFromCar(metricsStream, res, cidObj, req.params.path, blockFilename);
```

Comment:
The `path` param was removed a few days ago. Is this you adding it back, or just merging from an older commit? If the latter, could you update your PR to the latest?

Reply:
weird oops